TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -1846,19 +1272,8 @@ bool laneChangesAllowed
                                             maxNextCompatibleOuterSimilarIndex =
                                                 prevOuterSimilarLaneIndex - a;
 
-                                        if (extendedLogRouting) {
-                                            Log._DebugFormat(
-                                                "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, " +
-                                                "{2}, {3}): split outer lanes: " +
-                                                "minNextCompatibleOuterSimilarIndex={4}, " +
-                                                "maxNextCompatibleOuterSimilarIndex={5}",
-                                                prevSegmentId,
-                                                prevLaneIndex,
-                                                prevLaneId,
-                                                isNodeStartNodeOfPrevSegment,
-                                                minNextCompatibleOuterSimilarIndex,
-                                                maxNextCompatibleOuterSimilarIndex);
-                                        }
+                                        extendedLog?.Invoke(new { _ = "split outer lanes:", minNextCompatibleOuterSimilarIndex, maxNextCompatibleOuterSimilarIndex });
+
```

Comment:
here and in a few other places there is no need for empty line after log.

Reply:
To me, this enhances readability by visually separating logging code from program logic.

Reply:
i mean the next line is `}`. the problem is that if the log logic take too much space then less of the rest of the logic fit in one page.